packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2679,16 +2679,37 @@ curl -v -X POST \
 -H "Accept: application/json" \
 -H "Content-Type: application/json" \
 -H "Authorization: SSWS ${api_token}" \
-"https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR/lifecycle/reset_password?provider=FEDERATION&sendEmail=false"
+"https://${yourOktaDomain}/api/v1/users/{userId}/lifecycle/reset_password?provider=FEDERATION&sendEmail=false"
 ```
 
 ##### Response example
 
-
 ```json
 {}
 ```
 
+##### Request example (Convert a user to an Okta User)
+
+To convert a user to an Okta user, pass an empty value as the `provider` in the [Provider object](#provider-object). The `sendEmail`
+parameter must be false or omitted for this type of conversion.
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users/{userId}/lifecycle/reset_password?provider=&sendEmail=false"
```

Comment:
I think you can just remove the `provider=` part

Reply:
Hmm seems `sendEmail` isn't mandatory either.

Reply:
Yes, tested and you're correct. I'll make that update. It's basically the default call, but I think the example might help.